doc/source/examples/cem_iris.ipynb

```diff
@@ -18,6 +18,20 @@
     "This notebook requires the seaborn package for visualization which can be installed via pip:"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "To enable support for the Contrastive Explanation Method, you may need to run\n",
+    "    \n",
+    "```bash\n",
+    "pip install alibi[tensorflow]\n",
+    "```\n",
+    "\n",
+    "</div>"
```

Comment:
Same comment as above, can we make the box title be "Note" and the text part of the body?